### PR TITLE
Add CI node e2e and e2e test for containerd repo.

### PR DESCRIPTION
**Pull request by Random-Liu**
Setup node e2e and e2e CI test for containerd repo.

Depends on https://github.com/containerd/cri/pull/673.

/hold

### CODE REVIEW

File: jobs/config.json [hunk 1]
```diff
@@ -35,6 +35,70 @@
       "sig-node"
     ]
   },
+  "ci-containerd-build": {
+    "args": [
+      "${GOPATH}/src/github.com/containerd/cri/test/build-containerd.sh"
```
Comment (Random-Liu) [inline jobs/config.json]: Can I use environment variable here?

Comment (krzyzacy) [reply jobs/config.json]: just use `/go/src` I think

Comment (Random-Liu) [reply jobs/config.json]: Will do.

Comment (Random-Liu) [reply jobs/config.json]: Done

**Comment by Random-Liu** [comment]
/hold cancel
containerd/cri#673 is merged.

File: prow/config.yaml [hunk 2]
```diff
@@ -6653,6 +6653,66 @@ periodics:
...
+    - image: gcr.io/k8s-testimages/kubekins-e2e:v20180314-34ab1109a-experimental
+      args:
+      - "--repo=github.com/containerd/containerd=master"
+      - "--repo=github.com/containerd/cri=master"
+      - "--root=/go/src"
+      - "--upload=gs://kubernetes-jenkins/logs"
+
+- interval: 1h
+  agent: kubernetes
+  name: ci-containerd-e2e-gci-gce
```
Comment (krzyzacy) [inline prow/config.yaml]: what's the difference betweeh this and `ci-cri-containerd-e2e-gci-gce`?

Comment (Random-Liu) [reply prow/config.yaml]: This test is for `github.com/containerd/containerd` repository, not `github.com/containerd/cri`.

**Pull request merged**